### ISSUE TITLE
[Android] App crashes when started

### ISSUE THREAD

**Issue by superical** [evollu/react-native-fcm#267]
I've just followed through the README and had the configurations done. I've also copied the `google-services.json` file into the android/app folder. I don't have any JS code that uses react-native-fcm yet. There is only the library installed through npm and all those configurations. 

However, when I start the app through `react-native run-android`, the app crashes with the emulator showing that "Unfortunately, myApp has stopped." message.

I can complete the build successfully without errors in the console. Since the build process has no errors, I've no idea what else could have caused the app to crash.

Here's what I've in the dependencies in my build.gradle:

```
dependencies {
    compile project(':react-native-fcm')
    compile 'com.google.firebase:firebase-core:10.0.1'
    ...
}
```

I also have `apply plugin: "com.google.gms.google-services" `at the last line of the build.gradle file.

The rest of the configuration follows exactly as what was from the readme. 
What else can I try to fix this crashing issue after installing react-native-fcm?
And is there anyway I can see what was the underlying exception when that "Unfortunately, myApp has stopped" message appears?

Versions:
RN: 0.37
react-native-fcm: 2.5.6
Device: Android Nexus 6 Emulator

**Comment by evollu** [comment]
try use IDE and android studio
If I remember correctly, react native cli has an option show show logcat too.

**Comment by groke** [comment]
I ended up resolving this issue on my machine by removing the `apply plugin: 'com.google.gms.google-services' ` line from the end of android/app/build.gradle

**Comment by fmelp** [comment]
Hi there, I am having the exact same problem. App build is successful, but once app is installed on device it unexpectedly quits... Any suggestions?

**Comment by fmelp** [comment]
this is the error i get in Android Studio when app crashes
03-19 16:47:59.663 2033-5207/? W/ErrorProcessor: onFatalError, processing error from engine(4)
                                                 com.google.android.apps.gsa.shared.speech.a.g: Error reading from input stream
                                                     at com.google.android.apps.gsa.staticplugins.recognizer.i.a.a(SourceFile:342)
                                                     at com.google.android.apps.gsa.staticplugins.recognizer.i.a$1.run(SourceFile:1367)
                                                     at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
                                                     at java.util.concurrent.FutureTask.run(FutureTask.java:237)
                                                     at com.google.android.apps.gsa.shared.util.concurrent.a.ak.run(SourceFile:66)
                                                     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
                                                     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
                                                     at java.lang.Thread.run(Thread.java:761)
                                                     at com.google.android.apps.gsa.shared.util.concurrent.a.ad$1.run(SourceFile:85)
                                                  Caused by: com.google.android.apps.gsa.shared.exception.GsaIOException: Error code: 393238 | Buffer overflow, no available space.
                                                     at com.google.android.apps.gsa.speech.audio.Tee.g(SourceFile:2531)
                                                     at com.google.android.apps.gsa.speech.audio.ap.read(SourceFile:555)
                                                     at java.io.InputStream.read(InputStream.java:101)
                                                     at com.google.android.apps.gsa.speech.audio.al.run(SourceFile:362)
                                                     at com.google.android.apps.gsa.speech.audio.ak$1.run(SourceFile:471)
                                                     at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
                                                     at java.util.concurrent.FutureTask.run(FutureTask.java:237)
                                                     at com.google.android.apps.gsa.shared.util.concurrent.a.ak.run(SourceFile:66)
                                                     at com.google.android.apps.gsa.shared.util.concurrent.a.ax.run(SourceFile:139)
                                                     at com.google.android.apps.gsa.shared.util.concurrent.a.ax.run(SourceFile:139)
                                                     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133) 
                                                     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607) 
                                                     at java.lang.Thread.run(Thread.java:761) 
                                                     at com.google.android.apps.gsa.shared.util.concurrent.a.ad$1.run(SourceFile:85)

**Comment by evollu** [comment]
doesn't seem to be related to this repo. some speech recognizer threw error

**Comment by fdobre** [comment]
Hello, I'm having the same problem.
I have:
```
react-native-cli: 2.0.1
react-native: 0.39.2,
compile('com.google.android.gms:play-services-analytics:11.0.4')
compile('com.google.android.gms:play-services-ads:11.0.4')
compile('com.google.android.gms:play-services-gcm:11.0.4')
```
Did anyone find a workaround for this issue? I'm not using anything from speech.audio recording functionality.
I've fought with this issue for a full day and could not find the source cause or a workaround.

**Comment by evollu** [comment]
does this happen to all devices? try newer versions?

**Comment by fdobre** [comment]
Yes, that happened to all devices. Changed the app version and added <uses-permission android:name="android.permission.RECORD_AUDIO" /> in the manifest and the issue seem to be gone.

**Comment by johndrkurtcom** [comment]
I was having the same issue until found this article helped me to solve the issue... https://medium.com/@suchydan/how-to-solve-google-play-services-version-collision-in-gradle-dependencies-ef086ae5c75f

**Comment by anjan9635** [comment]
How it resolved
